### PR TITLE
US52_miniUS#54

### PR DESCRIPTION
**Pull request by grustia01**
Create new test for GET User

### CODE REVIEW

File: spec/webmock/user_spec.rb [hunk 1]
```diff
@@ -21,5 +21,34 @@
...
+      ]
+
+      stub_request(:get, 'http://localhost:3030/api/user')
+        .to_return(status: 200, body: body.to_json)
+
+      user_json = client.users
+      users = JSON.parse(user_json)
+
+      # Assert the response and test your code's behavior
+      expect(users[0]['email']).to eq('mb@example.com')
```
Comment (mrgutie8) [inline spec/webmock/user_spec.rb]: consider adding the following test to check that the request is not returning duplicate users
```ruby
expect(users.length).to eq(2)
```

**Review by mrgutie8 — APPROVED** [review]

File: spec/webmock/user_spec.rb (hunk 1 above)
Comment (grustia01) [reply spec/webmock/user_spec.rb]: considered, thanks for the recommendation!
